apps/website/src/pages/docs/fileupload.mdx

```diff
@@ -14,14 +14,22 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-fileupload' />
-
 <LiveExample src='FileUpload.main.tsx'>
   <AllExamples.FileUploadMainExample client:load />
 </LiveExample>
 
 File uploading is a common interaction most users do multiple times per day. File uploading may happen in various different ways depending on the context. While the overall design may differ from UI to UI, the basics always stay the same: let the user know they can upload content, display upload status progress, and confirm that the upload’s status upon completion.
 
+## Customizations
+
+### Custom File Upload Card
+
+Change standard file upload card by adding custom label and text.
```

Comment:
I think it would be useful to list all subcomponents we are exposing and adding an example with default subcomponents.

Reply:
I have in mind documentation, not code example: 
![image](https://github.com/iTwin/iTwinUI/assets/83585998/159529e8-6345-47d7-a9be-e5b9601cb176)

Reply:
The list of components can be found in react code.
You should move that list here or add another example in the section above.

Reply:
> You should move that list here

Remove Composition API section and just add the list in the customisations section

**OR**

> add another example in the section above

Create an example for composition API with standard subcomponents without customising them.

Section called "Composition API" with no example actually does not explain how to use subcomponents. Simply listing subcomponents does not make user understand it better. Those two sections are not linked together or anything. How would users know that these subcomponents are used in customisation example???